src/cdk/stepper/step-label.ts

```diff
@@ -12,5 +12,5 @@ import {Directive, TemplateRef} from '@angular/core';
   selector: '[cdkStepLabel]',
 })
 export class CdkStepLabel {
-  constructor(public template: TemplateRef<any>) { }
+  constructor(/** @docs-private */ public template: TemplateRef<any>) { }
```

Comment:
`_template`?

Reply:
Avoiding rename in case someone is using it